src/main/scala/beam/agentsim/agents/vehicles/BeamVehicle.scala

```diff
@@ -193,9 +224,9 @@ class BeamVehicle(
               chargingPoint,
               primaryFuelLevelInJoules,
               beamVehicleType.primaryFuelCapacityInJoule,
-              100.0,
-              100.0,
-              None
+              1e6, // todo JH this should be vehicle dependent
```

Comment:
Should this remain a todo for now? Maybe create an issue to track it more externally?

Reply:
agree